src/DotVVM.Framework/ResourceManagement/InlineScriptResource.cs

```diff
@@ -12,11 +12,6 @@ namespace DotVVM.Framework.ResourceManagement
     /// </summary>
     public class InlineScriptResource : ResourceBase
     {
-        [Obsolete("Code parameter is required, please provide it in the constructor.")]
-        public InlineScriptResource(ResourceRenderPosition renderPosition = ResourceRenderPosition.Body) : base(renderPosition)
```

Comment:
I have seen this one a lot of times, I'd also add it back.

Reply:
It's a v2.0 obsolete anyway. We should keep it.